templates/kubernetes/install.html

```diff
@@ -2,338 +2,634 @@
+      <p>
+        Canonical Kubernetes is pure upstream and works on any cloud, from bare metal to public and edge. Deploy single
+        node and multi-node clusters with Charmed Kubernetes and MicroK8s to support container orchestration, from
+        testing to production. Both distributions bring the latest innovations from the Kubernetes community within a
+        week of upstream release, allowing for time to learn, experiment and upskill.
+      </p>
+      <p>
+        <a class="p-button--positive" href="/kubernetes/contact-us">Get support for Kubernetes</a>
+      </p>
     </div>
     <div class="col-5 u-align--center u-hide--small">
       {{
-        image(
-        url="https://assets.ubuntu.com/v1/0822f908-kubernetes-instal.svg",
-        alt="Install Kubernetes",
-        width="140",
-        height="188",
-        hi_def=True,
-        loading="auto",
-        ) | safe
+      image(
+      url="https://assets.ubuntu.com/v1/0822f908-kubernetes-instal.svg",
+      alt="Install Kubernetes",
+      width="140",
+      height="188",
+      hi_def=True,
+      loading="auto",
+      ) | safe
       }}
     </div>
   </div>
 </section>
 
-<section class="p-strip--light is-bordered">
+<section class="p-strip is-bordered">
   <div class="row">
     <div class="col-8">
-      <h2>Deploy Kubernetes cluster</h2>
-      <h4>Where do you want to deploy Kubernetes?</h4>
+      <h2>Kubernetes deployment options</h2>
+      <p>Canonical helps you be successful with Kubernetes, starting from where you want to deploy it and how you intend
+        to use it.</p>
+      <p>Self-managed deployments give you greater control on Kubernetes infrastructure, from cloud to edge. Canonical
+        Managed Kubernetes provides a hands-off experience with great economics.</p>
     </div>
   </div>
   <div class="row u-equal-height">
-    <div class="col-6 p-card">
-      <h3><a href="#single-node">Single node with MicroK8s</a></h3>
-      <hr class="u-sv1" />
-      <div class="p-card__content">
-        <p>Install MicroK8s, the Linux snap that downloads  in seconds. Microk8s is lightweight and deploys all Kubernetes services natively on Ubuntu.  Ideal for:</p>
-        <ul class="p-list">
-          <li class="p-list__item is-ticked"><strong>Laptops:</strong> develop microservices locally</li>
-          <li class="p-list__item is-ticked"><strong>Workstations:</strong> develop and train machine learning models locally</li>
-          <li class="p-list__item is-ticked"><strong>CI pipelines:</strong> create ephemeral Kubernetes quickly for testing microservices</li>
-          <li class="p-list__item is-ticked"><strong>IoT devices:</strong> embed upgradeable Kubernetes in your Iot devices for easy evolution</li>
-          <li class="p-list__item is-ticked"><strong>Small edge clouds:</strong> create simple Kubernetes clusters for your edge clouds</li>
-        </ul>
-      </div>
+    <table class="p-table" style="margin-top: 1rem;">
```

Comment:
Could you try and add mobile card to make it easier to read on small screens.

Reply:
Added now, let me know what you think.